fhir-term-graph/kubernetes/k8ssandra.yaml

```diff
@@ -0,0 +1,29 @@
+cassandra:
+  version: "3.11.10"
+  cassandraLibDirVolume:
+    storageClass: ibmc-vpc-block-metro-10iops-tier
+    size: 5Gi
+  allowMultipleNodesPerWorker: true
+  heap:
+    size: 1G
+    newGenSize: 1G
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 2Gi
+    limits:
+      cpu: 1000m
+      memory: 2Gi
+  datacenters:
+  - name: dc1
+    size: 1
+    racks:
+    - name: default 
+kube-prometheus-stack:
+  enabled: false
+stargate:
+  enabled: true
+  replicas: 1
+  heapMB: 256
+  cpuReqMillicores: 200
+  cpuLimMillicores: 1000
```

Comment:
once again, not ideal... should go in different location.

Reply:
Moved.